src/Illuminate/Broadcasting/Broadcasters/PusherBroadcaster.php

```diff
@@ -26,6 +30,19 @@ class PusherBroadcaster extends Broadcaster
     public function __construct(Pusher $pusher)
     {
         $this->pusher = $pusher;
+
+        $this->pusher->set_logger(new class() {
+            public function log($message)
+            {
+                $app = Container::getInstance();
```

Comment:
Bad practice to do this. Instead, inject the psr logged into the constructor.

Reply:
nevermind

Reply:
I really need this one @GrahamCampbell can you add this feature?